mantidproject/mantid
# Reflectometry algorithm output workspaces

**Description of work**

Added `Debug` property to the reflectometry algorithms. Setting this property to `true` makes the algorithms to output additional workspaces.

**To test:**

Run the algorithms with `Debug` set and unset. Check that the workspaces are created as described in the issue.


Fixes #22259


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
